cruise/CruiseInformation.csv

```diff
@@ -1,54 +1,75 @@
 CUID,ShipName,cruiseStartDateTime,cruiseStopDateTime,notes
 UNRECOVERED,N/A,2009-09-01T00:00:00,2040-09-30T00:00:00,this is a placeholder for platforms/instruments that are not recovered
-TN-299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata)
-MV-1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1
-KN-214,R/V Knorr,2013-11-20T18:00:00,2013-11-26T14:00:00,Pioneer 1
+TN-299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata) #TO BE DELETED
+TN299,R/V Thomas G. Thompson,2013-06-30T00:00:00,2013-08-23T23:59:59,Cabled (rvdata)
+MV-1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1 #TO BE DELETED
+MV1309,R/V Melville,2013-07-15T00:00:00,2013-07-30T23:59:59,Papa 1
+KN-214,R/V Knorr,2013-11-20T18:00:00,2013-11-26T14:00:00,Pioneer 1 #TO BE DELETED
+KN214,R/V Knorr,2013-11-20T18:00:00,2013-11-26T14:00:00,Pioneer 1
 TY-1402,CCGS Tully,2014-02-20T00:00:00,2014-02-20T00:00:00,Papa Glider Operations
 OC1404B,R/V Oceanus,2014-04-15T00:00:00,2014-04-21T23:59:59,Endurance 1 (rvdata)

```

Comment:
This should be after KN217 to be chronological.

Reply:
done

Reply:
Looks good